corehq/apps/style/templates/style/includes/modal_report_issue.html

```diff
@@ -72,9 +72,23 @@ <h4 class="modal-title">{% trans "Report an Issue with CommCare HQ" %}</h4>
                         </div>
                         {% endif %}
 
+                        {% if request.couch_user.is_domain_admin %}
```

Comment:
[From the [spec](https://docs.google.com/document/d/12r6fkzpy5vlVj3_i0qwcBzu8VeizJDIAPYVyDpQ4uzw/edit#)] If the project-description already exists, this field should be hidden. It doesn't look like this block is handling that?